content/verification/veteran-verification/release-notes/2023-05-18

```diff
@@ -0,0 +1,26 @@
+We added a new version(V2) of the Veteran Verification API to the developer portal, and the API has been renamed to 'Veteran Service History and Eligibility API'.
+
+The following enhancements and new features are available in V2:  
+
+All of the API's endpoints can now be accessed using either Authorization Code Flow OAuth or Client Credentials Grant (CCG) OAuth.   
```

Comment:
Will all of these be listed as bullet points in the UI, up to 'each individual rating'?